polycommit/src/marlin_pc/data_structures.rs

```diff
@@ -234,6 +241,19 @@ pub struct Commitment<E: PairingEngine> {
 }
 impl_bytes!(Commitment);
 
+impl<E: PairingEngine> ToMinimalBitRepresentation for Commitment<E> {
```

Comment:
This is partly why this trait is needed, as we need to convert the commitment, but Marlin does not have access to the specific struct because Marlin is parameterized by a generic polynomial commitment.